.github/workflows/ci.yml

```diff
@@ -78,5 +78,6 @@ jobs:
         uses: ./.github/actions/test
         with:
           toolchain: nightly
+          target: "x86_64-unknown-linux-gnu"
```

Comment:
Why was this line added?

Reply:
https://github.com/shepmaster/jetscii/pull/57/commits/7800819417528a5183a2df1db5c8f59e45ffa91d I was trying to fix the ci run for asan

Reply:
Ah. I'm using the `target` field as an implicit "should I cross build" flag, so that triggered `cross`, but it looks like it failed for the same root cause. I wonder if splitting that apart would help the problem.

I wonder if you can hack out the cross build CI logic and see if the explicit target helps then. I might do a parallel branch where I upgrade off of `actions-rs` and maybe detangle the `target` and `cross` concepts.